src/client-private.js

```diff
@@ -88,10 +88,13 @@ function createMediaElement (identifier) {
 
 function cleanupMediaElement (identifier) {
   const elem = getMediaElement(identifier);
-  if (elem) {
-    this._log('info', 'Removing media element', { identifier });
-    elem.parentNode.removeChild(elem);
+  if (!elem) {
+    this._log('info', 'failed to find audio element for removal', { identifier });
```

Comment:
Is the reason you are using `'info'` here instead of `'warn'` because it isn't really dangerous if the element if already removed? That is my assumption, but I just wanted to be sure.

Reply:
I was kind of in the air on this one. It's not really dangerous, you'd just end up with a bunch of audio elements on the page. The audio will be dead either way.

Reply:
How can you get into the condition where the identifier doesn't match up with an audio element?